README.md

```diff
@@ -48,7 +48,130 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefact
 )
 ```
 
-## Setting up the accessibility test folder
+## Plugin Setup
+
+There are two different approaches to writing accessibility tests, you may choose the approach that
+suites your team best.
+
+## Automatic accessibility testing 
+
+Automatic accessibility testing allows you to write tests via a single spec, it can do this by
+automatically scanning through your project for views.
```

Comment:
rather than views, what about "page templates"? I feel like we're looking inside the views package for a specific type of view, a page template